TODO.md

```diff
@@ -0,0 +1,24 @@
+# Confirm Licensing and Copyright
```

Comment:
Oh didn't see this file before. Some of this is specific to us, so we might want to strip it out?  If we want to keep this file as the current state of the project, we should point the README at this file